crates/sound/src/stream.rs

```diff
@@ -8,6 +8,7 @@ use vm_memory::{Le32, Le64};
 
 use crate::{virtio_sound::*, IOMessage, SUPPORTED_FORMATS, SUPPORTED_RATES};
 
+pub const NR_STREAMS: usize = 1;
```

Comment:
Any reason this was added here if it is only used by the pipewire backend?

Reply:
This can be removed.